test/PrefilterExpressionIndexTest.cpp

```diff
@@ -84,7 +84,8 @@ class PrefilterExpressionOnMetadataTest : public ::testing::Test {
   const BlockMetadata b24 =
       makeBlock(DateId(DateParser, "2024-10-08"), BlankNodeId(10));
 
-  // All blocks that contain mixed (ValueId) types over column 0
+  // All blocks that contain mixed (ValueId) types over column 0,
+  // or possibly incomplete ones.
```

Comment:
Is there already an example where the first and the last block are not matching (you need one with the match and one without for the coverage.